### ISSUE TITLE
Notebooks not using same seed as experiments, affects reproducibibilty across notebooks and experiments

### ISSUE THREAD

**Issue by yalaudah** [microsoft/seismic-deeplearning#244]
In the notebooks, the seeds are defined manually (see below) instead of reading from the config file like the experiments. This should be fixed to get reproducible results across notebooks and experiments. 

```python 
np.random.seed(seed=1234)

import torch

torch.backends.cudnn.benchmark = True
torch.manual_seed(1234)
if torch.cuda.is_available():
    torch.cuda.manual_seed_all(1234)
```

**Comment by yalaudah** [comment]
Fixed in #247

**Comment by yalaudah** [comment]
Fixed in #247
